tendrl/commons/objects/gluster_peer/atoms/gluster_server_peer_check/__init__.py

```diff
@@ -0,0 +1,91 @@
+import etcd
+import json
+import socket
+
+from tendrl.commons import objects
+from tendrl.commons.utils import cmd_utils
+from tendrl.commons.utils import etcd_utils
+from tendrl.commons.utils import log_utils as logger
+
+
+class GlusterServerPeerError(Exception):
```

Comment:
Why pass? No specific error messages?

Reply:
If glusterd is down then gluster pool list command will give an error message like gluster daemon is down. So i have created a user-defined exception to raise an exception with a proper exception message. 

And single try-catch block will catch an exception and log the error message into a socket. So we don't need different try-catch block.